vercel/turbo
# build: Update `swc_core` to `v0.90.29`

### Description

Update SWC crates.


1. To keep in sync
2. Prepare usage of source map range mappings. https://github.com/getsentry/rust-sourcemap/pull/77


### Testing Instructions

See next.js counterpart.

https://github.com/vercel/next.js/pull/63790

Closes PACK-2861
